backend/Utils/EmailUtils.ts

```diff
@@ -2,9 +2,32 @@ import google from "googleapis";
 import path from "path";
 import { encode } from "js-base64";
 import dotenv from "dotenv";
+import jwt from "jsonwebtoken";
 
-async function makeEmail({ to, from, subject }) {
-  const message = `This is a test email for reset password.`;
+dotenv.config();
+const { sign, verify } = jwt;
+
+const baseUrl = process.env.PASSWORD_RESET_BASE_URL || "https://localhost:7600";
+
+async function makeEmail({ to, from, subject }, token) {
+  const message = `
+  <div style="width: 60vw; margin: 4rem auto auto auto; color: #403F3D;
+  ">
```

Comment:
A little unusual to put a new line inside of the quotes of a `style` html attribute. Let's consolidate this to just one line.